src/applications/claims-status/components/ClaimsDecision.jsx

```diff
@@ -16,19 +16,13 @@ const ClaimsDecision = ({ completedDate, showClaimLettersLink }) => (
       </h3>
       {showClaimLettersLink ? (
         <p>
-          You can download your decision letter online now. You can also get
-          other letters related to your claims.
+          You can download your decision letter online now. We also mailed you
+          this letter.
         </p>
       ) : (
         <p>We finished reviewing your claim and a decision has been made.</p>
       )}
 
-      <p>
-        We’ll also send you a copy of your decision letter by mail. It should
-        arrive within 10 days after the date we closed your claim, but it may
-        take longer.
-      </p>
-
```

Comment:
Entire paragraph remove and text regarding mailing the letter was added to line 19. I reviewed these changes with @mistymg over Zoom.